### ISSUE TITLE
Website Domain

### ISSUE THREAD

**Issue by jdhaines** [aiaa-deic/computation-ai#12]
You'll need to decide on a domain for the built website:

The default is: https://aiaa-deic.github.io/computation-ai/

I've already got this one ready to go: https://compai.aiaadeic.org/

Or you can pick any other subdomain at aiaadeic.org or we can get another domain if you like.  Just let me know and I'll configure it.

**Comment by arthurrge** [comment]
Your ready-to-go looks great! Thank you!

**Comment by jdhaines** [comment]
Wait, I'm confused... neither of those links are set up and working yet.  The first only shows the readme, which isn't great.  I'll make sure yours shows the actual website like this one: https://devsecops.aiaadeic.org/

Just need to know where to host it...

**Comment by arthurrge** [comment]
I'm confused too now. I like what you have at https://devsecops.aiaadeic.org/ and the likewise for compai would be great - I can look into how to replicate what you have, or if there is an overlay that can be copied, that is even better.

**Comment by arthurrge** [comment]
Catching up on this - there seem to be errors deploying GitHub pages - and I wasn't sure what needed to be done to have the https://compai.aiaadeic.org/ linked up properly.

**Comment by jdhaines** [comment]
Looks like we're all good.  I didn't flip this switch once you got the PR approved.  Actions didn't have permissions to make the push back to the gh-pages branch.  Once that was solved the action completed successfully.  

![image](https://github.com/aiaa-deic/computation-ai/assets/3834260/6731f9ca-8cbd-4e1b-b461-d31f2942ace1)

Then I changed the github pages branch from `main` to `gh-pages`.  On main it just deploys the readme.  

![image](https://github.com/aiaa-deic/computation-ai/assets/3834260/ec423ce4-4e40-4d0c-ab39-0a0da31aac2b)

Essentially, every time you make a push it'll run the github action, build the site, and force push a copy to the `gh-pages` branch.  That's the branch that GitHub Pages watches.  The domain is now working and HTTPS is enforced.  Should be good to go!  Feel free to close this if you agree.

![image](https://github.com/aiaa-deic/computation-ai/assets/3834260/255f0ce6-33d3-4a28-8fa0-3e6cd96971c5)
